UnityProject/Assets/Scripts/Health/Living/DNABloodTypeGen.cs

```diff
@@ -0,0 +1,50 @@
+using Random = UnityEngine.Random;
+
+[System.Serializable]
+public class DNAandBloodType
+{
+	public string DNAString;
+	public string BloodType;
+
+	enum bloodtypes { oneg, opos, bneg, bpos, aneg, apos, abneg, abpos}
```

Comment:
It would be best to declare this outside the DNAandBloodType class so that it is accessible everywhere. Just move this line so it is outside the class and make it public